### ISSUE TITLE
User-portal shows an error when user email verification on update is enabled

### ISSUE THREAD

**Issue by rksk** [wso2/product-is#10064]
**Describe the issue:**
When "Enable user email verification on update" is enabled from the resident IDP, the user portal email update shows both success and failure messages like below.
![claim update issue](https://user-images.githubusercontent.com/719073/97114365-1ee55f00-1716-11eb-816e-46f14360b05b.png)

The user email is not yet updated and it will be updated only after the link in the confirmation email is clicked.

**How to reproduce:**
- Enable "Enable user email verification on update" from the resident IDP
- Login to user-portal and try to change the email address

**Expected behavior:**
The user should be informed that a confirmation email has been sent out and he/she needs to complete the verification in order to change take effect.

**Environment information**
 - Product Version: IS 5.10.0

**Comment by dewniMW** [comment]
Related to: https://github.com/wso2/product-is/issues/9046

**Comment by rksk** [comment]
Closing since this is a duplicate of #9046
